src/domains/job/common/components/JobInfiniteScroll.jsx

```
@@ -0,0 +1,74 @@
+      const response = await fetcher(lastId, order)
+      let newPosts = response?.[dataKey] || []
+      const newLastId = response?.lastId
+
+      setPosts((prev) => {
+        const existingIds = new Set(prev.map((post) => post.id))
+        const filtered = newPosts.filter((post) => !existingIds.has(post.id))
+        return [...prev, ...filtered]
+      })
+      setLastId(newLastId)
+      setHasMore(newPosts.length > 0)
+    } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(no-unused-vars)